src/main/java/com/bankxpvalue/BankXpValuePlugin.java

```diff
@@ -82,69 +79,45 @@ public void onMenuEntryAdded(MenuEntryAdded event){
             return;
         }
 
-        MenuEntry[] entries = client.getMenuEntries();
-
         if (config.showTutorial()){
```

Comment:
Code looks great, but the only thing is is that if you enable the tutorial while you have the overlay position saved outside of the center, the tutorial overlay looks bugged (its position is fixed too).

I'm not sure why anyone would enable the tutorial after already using the plugin, but just to keep everything bug-free, perhaps we should toggle the `keepFixed` option when the user enables the tutorial. We don't even have to keep it enabled, just turning it on and off would reset the position of the overlay to the center and match the tutorial text. What do you think?

<img src="https://user-images.githubusercontent.com/47870467/149426344-6c4b6998-5407-44ea-a520-ccc9acef8ce0.png" width="250">

Reply:
Hmmm....I'll look into it, but I'm wondering if it would just be better to move the tutorial overlay.

Reply:
Another option is just to not let the users turn on the tutorial again by hiding that config setting :P

Reply:
Though I guess that doesnt solve the issue completely because on first run it won't be centered...will it? I didn't try that case. What happens on first run? Maybe we should default `keepFixed` to `true`

Reply:
I tried the dynamic tutorial overlay implementation when I first wrote the plugin, and I couldn't figure it out. I think that was part of the reason why I kept the banked xp overlay fixed in the center. I mean, if you want to try to figure it out yourself, be my guest lol. If not, I honestly think that just hiding the tutorial config setting after first use is the best way to go. I'll accept that route.

Reply:
Cool, I'll give it a shot tonight when I have some time!

Reply:
I got it working, it follows the overlay around when you drag it even :)

Reply:
Nice!!!! I just played around with it and it looks great! 👏🏼

My only concern is that the overlay does not go back to the default center position when you reset the plugin settings -- it stays where you moved it. Can you resolve this?

https://user-images.githubusercontent.com/47870467/149635264-6e5949c4-e85a-4ab4-b4b7-64c00efd4522.mov

Reply:
Regarding this, the location that is saved is namespaced to runelite so resetting it wont effect it. And I'm not sure it's possible to hook that event. It looks like this `runelite.BankXpValueOverlay_preferredLocation=157\:69`

Reply:
Ah I see I see. This is fine then.